cmd/clean-main/main.go

```diff
@@ -31,11 +30,10 @@ func main() {
 	flag.StringVar(&password, "password", "", "NSX ALB Controller authentication password")
 	flag.StringVar(&albCtrlCert, "cacert", "", "NSX ALB Controller authentication certificate")
 	flag.StringVar(&clusterID, "cluster-id", "", "AKO cluster ID")
-	flag.StringVar(&supervisorID, "supervisor-id", "", "Supervisor ID")
```

Comment:
will wcpsvc be sending supervisorid in clusterid field if it wants to set?

Reply:
yes, wcpsvc will send either clusterID or SupervisorID in the clusterID field